### DISCUSSION TITLE
Does the Chocolatey package still exist?

### DISCUSSION THREAD

**Discussion by TomLucidor** [LeDragoX/Win-Debloat-Tools#151] [Q&A] [answered]
Would like to see this as the alternative to https://github.com/simeononsecurity/Windows-Optimize-Debloat
Also gonna check the following GUIs out:
- https://github.com/farag2/Sophia-Script-for-Windows
- https://github.com/undergroundwires/privacy.sexy

**Comment by LeDragoX** [comment, marked as answer]
> Does the Chocolatey package still exist?

Huh? What do you mean?

> Would like to see this as the alternative to [...]

I only kept one debloater alternative because it was from CTT whom I learnt PowerShell, but i know there are other GREAT options among the GitHub repos. But i don't imagine myself putting a summary of scripts on the README file, sorry.
